setup-py-project.md

```diff
@@ -0,0 +1,62 @@
+### 1. Add GitHub workflow
+The workflow [file](https://github.com/Flow-Launcher/plugin-samples/blob/master/HelloWorldPython/.github/workflows/Publish%20Release.yml) will help build and deploy your project, it does the following things:
+1. `workflow_dispatch:` gives you the option to manually run your workflow from the Actions section of your project
+2. On pushes to main, it will kick off the workflow but ignore the push if it's only changes made to the workflow file.
+```yml
+push:
```

Comment:
The CI trigger should a commit with "$(flag): $(version_number)" (ex. "RLS: v0.1.0).

This commit only changes the `__version__` number in `plugin.settings`.
And the rest of things could be changed automatically after the publish/release CI workflow.
We don't need to care about the version in other places (ex. plugin.json) is right or not.

Reply:
Do you mean?
```
  push:
    branches: [ main ]
    tags:
      - 'v*'
```

Reply:
https://github.com/jjw24/Wox.Plugin.Runner/blob/master/.github/workflows/publish.yml
Why not try automatically creating tag based on plugin.json, which has been used in the Runner plugin?